coral/functions/licence_number_function.py

```diff
@@ -0,0 +1,287 @@
+from arches.app.search.elasticsearch_dsl_builder import Query
+from arches.app.search.search_engine_factory import SearchEngineInstance as se
+from arches.app.views.search import RESOURCES_INDEX
+from arches.app.functions.base import BaseFunction
+from arches.app.models.resource import Resource
+from arches.app.models.tile import Tile
+import datetime
+import json
+from django.db.models import Max
+
+LICENCE_GRAPH_ID = "cc5da227-24e7-4088-bb83-a564c4331efd"
+ACTIVITY_GRAPH_ID = ""
+LICENCE_NAME_NODEGROUP = "59d65ec0-48b9-11ee-84da-0242ac140007"
+LICENCE_NAME_NODE = "59d6676c-48b9-11ee-84da-0242ac140007"
+
+LICENCE_NUMBER_NODEGROUP = "6de3741e-c502-11ee-86cf-0242ac180006"
+LICENCE_NUMBER_NODE = "9a9e198c-c502-11ee-af34-0242ac180006"
+
+SYSTEM_REF_NODEGROUP = "991c3c74-48b6-11ee-85af-0242ac140007"
+SYSTEM_REF_RESOURCE_ID_NODE = "991c49b2-48b6-11ee-85af-0242ac140007"
+
+STATUS_NODEGROUP = "4f0f655c-48cf-11ee-8e4e-0242ac140007"
+STATUS_NODE = "a79fedae-bad5-11ee-900d-0242ac180006"
+STATUS_FINAL_VALUE = "8c454982-c470-437d-a9c6-87460b07b3d9"
+
+CUR_D_DECISION_NODEGROUP = "c9f504b4-c42d-11ee-94bf-0242ac180006"
+CUR_D_DECISION_NODE = "2a5151f0-c42e-11ee-94bf-0242ac180006"
+CUR_D_DECISION_APPROVED_VALUE = "0c888ace-b068-470a-91cb-9e5f57c660b4"
+
+ASSOCIATED_ACTIVIY_NODEGROUP = "a9f53f00-48b6-11ee-85af-0242ac140007"
```

Comment:
Added ASSOCIATED_ACTIVIY_NODEGROUP and ASSOCIATED_LICENCE_NODEGROUP

I duplicated them as ASSOCIATED_ACTIVIY_NODE and ASSOCIATED_LICENCE_NODE for the sake of the Tile logic looking the same for these.